### PR TITLE
Thr 18 feature h5p editor

### PR DESCRIPTION
**Pull request by MajedAlaitwniCap**
# Description

## Links to Tickets or other pull requests
PR schulcloud-server: https://github.com/hpi-schul-cloud/schulcloud-server/pull/4337
PR nuxt-client : https://github.com/hpi-schul-cloud/nuxt-client/pull/2748
PR: schulcloud-client: https://github.com/hpi-schul-cloud/schulcloud-client/pull/3277

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/host_vars/default_host/cfg.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FEATURE_H5P_EDITOR_ENABLED: "false"
+FEATURE_H5P_EDITOR_ENABLED: "true"
```
Comment (CeEv) [inline ansible/host_vars/default_host/cfg.yml:1]: My last info is that h5p should only rollout on thr. If this is correct than it must be changed.

**Review by CeEv — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/config.yml [hunk 2]
```diff
@@ -85,6 +85,16 @@ configuration_all:
     server: false
     client: false
     nuxtclient: true
+  H5P_IMG_SRC_URLS:
+    value: "https://h5p.org/"
```
Comment (Loki-Afro) [inline ansible/group_vars/all/config.yml:89]: there is still no solution for that one ..

**Review by Loki-Afro — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/config.yml (hunk 2 above)
Comment (casparneumann-cap) [reply ansible/group_vars/all/config.yml:89]: This will be solved in correspondence with Hannes Hensel via implementing a proxy.

**Review by CeEv — APPROVED** [review]

File: ansible/group_vars/all/config.yml (hunk 2 above)
Comment (CeEv) [reply ansible/group_vars/all/config.yml:89]: The proxy is already implemented, or a open task with existing ticket?

Comment (casparneumann-cap) [reply ansible/group_vars/all/config.yml:89]: Der Proxy wird aktuell vom SRE Team bearbeitet. Hauptansprechpartner hierzu sind Lars Dahlke & Dimitrij Pinneker

**Review by mamutmk5 — COMMENTED** [review]
Befor it go to main the ingress definitions must addet to the new ingress conzept.
It's mean that the ingress definiton for h5p-staticfiles-server must go to the ansible role and files at https://github.com/hpi-schul-cloud/h5p-staticfiles-server
Not more https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/host_vars/default_host/x_ingress.yml
https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/all/x_ingress.yml#L98-L107

**Review by Loki-Afro — APPROVED** [review]
not approving for myself but entwicklungsleitung

**Pull request merged**